README.md

```diff
@@ -1,37 +1,123 @@
-#WP PHP Console
+# === WP PHP Console ===
```

Comment:
this file is now the source of README.txt
only a few changes a required, e.g. removing the # github markdown in wordpress titles